site/plugins/site/extensions/tags.php

```diff
@@ -48,16 +48,18 @@
 ];
 
 /**
- * (screencast: https://www.youtube.com/watch?v=EDVYjxWMecc title: How to install Kirby in 5 minutes)
+ * (screencast: https://www.youtube.com/watch?v=EDVYjxWMecc cover: youtube.jpg title: How to install Kirby in 5 minutes)
  */
 $tags['screencast'] = [
 	'attr' => [
+		'cover',
 		'title',
 		'text',
 	],
 	'html' => function ($tag) {
 		return snippet('kirbytext/screencast', [
 			'url'   => $tag->value,
+			'cover' => $tag->file($tag->cover),
```

Comment:
```suggestion
			'cover' => $tag->cover ? $tag->file($tag->cover) : $tag->file('youtube.jpg'),
```

`Suggestion:` We could keep our rest of the code DRY-er if we add this default.